### ISSUE TITLE
Basic Zod Example does not compile

### ISSUE THREAD

**Issue by davidmdm** [vriad/zod#25]
I tried to use Zod and cannot get even the most basic example from the documentation to compile.

Here is the code:

```
import zod from 'zod';
zod.object({
  name: zod.string(),
  neutered: zod.boolean(),
});
```

Using typescript 3.8.3 with tsConfig:

```
{
  "compilerOptions": {},
  "include": ["src/**/*"]
}
```

Finally the message typescript provides:

```
node_modules/zod/lib/types/object.d.ts:46:43 - error TS2344: Type '{ [k in Exclude<keyof Params, "strict">]: Params[k]; } & { strict: false; }' does not satisfy the constraint 'ZodRawShape'.
  Property 'strict' is incompatible with index signature.
    Type 'false' is not assignable to type 'ZodAny'.

46     nonstrict: () => ZodObject<T, Flatten<{ [k in Exclude<keyof Params, "strict">]: Params[k]; } & {
                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
47         strict: false;
   ~~~~~~~~~~~~~~~~~~~~~~
48     }>>;
   ~~~~~

```

**Comment by vriad** [comment]
Hm, sorry about that. I implemented a new feature using TypeScript 3.8.3 and didn't realize I used functionality that wasn't compatible with earlier versions. I'm figuring out how to re-implement this in a way that's compatible with TS 3.0+.

In the meantime this will be fixed if you upgrade to TypeScript 3.8. Let me know if that's not the case.

**Comment by vriad** [comment]
Sorry, just realized you said you're using 3.8.3. Investigating now.

**Comment by vriad** [comment]
Should be fixed in zod@1.0.14.

Also you should import `zod` like so, because there's no default export:

`import * as z from "zod";`
